insurance.go

```diff
@@ -82,6 +79,37 @@ func (c *Client) ListInsurancesWithContext(ctx context.Context, opts *ListOption
 	return
 }
 
+// GetNextInsurancePage returns the next page of insurance records
+func (c *Client) GetNextInsurancePage(collection *ListInsurancesResult) (out *ListInsurancesResult, err error) {
+	return c.GetNextInsurancePageWithContext(context.Background(), collection)
+}
+
+// GetNextInsurancePageWithPageSize returns the next page of insurance records with a specific page size
+func (c *Client) GetNextInsurancePageWithPageSize(collection *ListInsurancesResult, pageSize int) (out *ListInsurancesResult, err error) {
+	return c.GetNextInsurancePageWithPageSizeWithContext(context.Background(), collection, pageSize)
+}
+
+// GetNextInsurancePageWithContext performs the same operation as GetNextInsurancePage, but
+// allows specifying a context that can interrupt the request.
+func (c *Client) GetNextInsurancePageWithContext(ctx context.Context, collection *ListInsurancesResult) (out *ListInsurancesResult, err error) {
+	return c.GetNextInsurancePageWithPageSizeWithContext(ctx, collection, 0)
+}
+
+// GetNextInsurancePageWithPageSizeWithContext performs the same operation as GetNextInsurancePageWithPageSize, but
+// allows specifying a context that can interrupt the request.
+func (c *Client) GetNextInsurancePageWithPageSizeWithContext(ctx context.Context, collection *ListInsurancesResult, pageSize int) (out *ListInsurancesResult, err error) {
+	if collection.Insurances == nil || len(collection.Insurances) == 0 {
```

Comment:
```suggestion
	if len(collection.Insurances) == 0 {
```

Reply:
`len()` will return `0` if an array/slice is `nil`

Reply:
Nice, thanks!